lib/jxl/enc_fast_lossless.cc

```diff
@@ -3516,30 +3594,37 @@ bool detect_palette(const unsigned char* r, size_t width,
 }
 
 template <typename BitDepth>
-JxlFastLosslessFrameState* LLEnc(const unsigned char* rgba, size_t width,
-                                 size_t stride, size_t height,
-                                 BitDepth bitdepth, size_t nb_chans,
-                                 bool big_endian, int effort,
-                                 void* runner_opaque,
-                                 FJxlParallelRunner runner) {
+JxlFastLosslessFrameState* LLPrepare(JxlChunkedFrameInputSource input,
```

Comment:
Is there a way to know at this point whether the *input* is streamed?

Reply:
The caller should know, I added a TODO for plumbing that it.